packages/calypso-products/src/plans-list.tsx

```diff
@@ -881,6 +883,14 @@ const getPlanEcommerceDetails = (): IncompleteWPcomPlan => ( {
 		},
 	} ),
 } );
+const getPlanWooExpressMediumDetails = (): IncompleteWPcomPlan => ( {
+	...getPlanEcommerceDetails(),
+	getTitle: () => i18n.translate( 'Woo Express Performance' ),
+	getTagline: () =>
+		i18n.translate(
+			'Learn more about everything included with Woo Express Performance and take advantage of its powerful marketplace features.'
```

Comment:
This doesn't seem right to me -- this is meant to be a short summary of what the plan offers. The value is used as the summary in the plans grid below the plan name/title.

Reply:
This is the same as the Commerce plan; I've just replaced the plan name. Maybe you thought it was `getPlanTagline()` ? 

(Which I kept the same: `Sell products and process payments with an online store.`)

Reply:
🤦 You’re right! I didn’t realize that there were two different taglines!